.github/workflows/mcvs-pr-validation.yml

```diff
@@ -17,3 +17,5 @@ jobs:
     steps:
       - uses: actions/checkout@v4.1.7
       - uses: ./
+        with:
+          gh_issue_check: true
```

Comment:
Could you test this branch in the mcvs-determiner?

Reply:
It's tested => https://github.com/schubergphilis/mcvs-determiner/pull/142
But as I can see this repository uses the same code so it applies here as well.